server/controllers/session.controller.js

```diff
@@ -7,6 +7,10 @@ export function createSession(req, res, next) {
       return res.status(401).send({ message: 'Invalid username or password.' });
     }
 
+    if (req.user) {
```

Comment:
this will redirect the API, which is not what we want. the only file you should need to change is `server/routes/server.routes.js`.